example.js

```diff
@@ -0,0 +1,35 @@
+// Require the necessary discord.js classes
+const { Client, Intents } = require('discord.js');
+const { token } = require('./config.json');
+const roverify = require("roverify");
+
+// Create a new client instance
+const client = new Client({ intents: 513 });
+// Intents Calc: https://ziad87.net/intents/
+
+// When the client is ready, run this code (only once)
+client.once('ready', () => {
+	console.log('Ready!');
+});
+
+client.on('messageCreate', async(msg) => {
```

Comment:
Seeing as this is async, you can use await instead of then.

Reply:
I'll start making all changes :)